bloom-housing/bloom
# fix: monthly rent unit seed data

## Description

Seed data bug was introduced in https://github.com/metrotranscom/doorway/pull/760. toPercision makes a string with `e+` which causes a problem in the partners portal. Switching to toFixed instead to limit decimals but not introducing other string characters.

## How Can This Be Tested/Reviewed?

Run `yarn setup`. Check `units` table in the db or run partners portal and check the seeded listing units.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
